public/resources/js/directives/api-list.js

```diff
@@ -39,6 +39,7 @@ angularIO.directive('apiList', function () {
 
       $ctrl.apiTypes = [
         { cssClass: 'stable', title: 'Only Stable', matches: ['stable']},
+        { cssClass: 'experimental', title: 'Experimental', matches: ['experimental']},
```

Comment:
I intentionally didn't make this "Only Experimental" because that's too long and gets clipped in the dropdown menu.